viewing_party/party.py

```diff
@@ -1,23 +1,123 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    print("***************************")
+    new_movie = {
+        'title': title,
+        'genre': genre,
+        'rating' : rating
+    }
+    for value in new_movie.values():
+        if value is None:
+            return None
+    return new_movie
 
+def add_to_watchlist(user_data, movie):
+    print("*******************************")
+    if "watchlist" in user_data:
+        user_data["watchlist"].append(movie)
+    return user_data
+
+def add_to_watched(user_data, movie):
+    print("*******************************")
+    if "watched" in user_data:
+        user_data["watched"].append(movie)
+    return user_data
+
+def watch_movie(user_data, movie_title):
+    for movie in user_data["watchlist"]:

```

Comment:
Note, while this may work here, it's considered dangerous practice to modify the content of a data structure that you're looping over.  Especially in more complex code, this can open you up to complex errors. A simple alternative here could be to save a copy of what you get from movie in user_data["watchlist"] and iterate over that, so that as you modify user_data["watchlist"], there's no risk.